### ISSUE TITLE
Windows Modern App

### ISSUE THREAD

**Issue by aneesh-neelam** [VinayGupta23/VITacademics_Windows#6]
Hey, where's the Windows Modern app? 

I told you, I want to see this on my computer as well.

**Comment by VinayGupta23** [comment]
Will start work on this real soon. Yeah...

**Comment by VinayGupta23** [comment]
It should, as Windows 8.1 Silverlight. (presumably like how it did with windows 8 and windows 8.1)
Once I download VS 2015, Ill be making it native to Windows 10 (universal).

**Comment by aneesh-neelam** [comment]
Silverlight?

**Comment by VinayGupta23** [comment]
When windows 8.1 released, windows 8 apps were still compatible, but they ran as "Windows 8 Silverlight" on the Windows 8.1 platform.

So I suppose the same will be done in the case of Windows 8.1 apps on Windows 10.

**Comment by aneesh-neelam** [comment]
That makes no sense. 

I have Windows 10 preview on my computer, I have no idea what you're talking about.

**Comment by VinayGupta23** [comment]
Its done internally.
Nothing that end-users are told about.

What I'm saying is when I upgraded to windows 8.1, any apps that are built and deployed ONLY for windows 8 still work without a fuss on 8.1.
That was done for compatibility so every developer can take time to upgrade.

That's why I could use the old VITacademics on windows 8.1 though it was released specifically for windows 8 only.

So I suppose they should do the same. Ill confirm it all the same at the Microsoft site.
When I create a new project in VS, I have options to create a windows 8.1 only, or create a windows 8 Silverlight app that will run on both windows 8 and 8.1. (But that of course will have restrictions.)

Sent from my Windows Phone

-----Original Message-----
